src/raftstore/coprocessor/properties.rs

```diff
@@ -383,7 +383,7 @@ impl TablePropertiesCollector for SizePropertiesCollector {
         self.index_handle.size += size;
         self.index_handle.offset += size as u64;
         // Add the start key for convenience.
-        if self.last_key.is_empty() || self.index_handle.size >= PROP_SIZE_INDEX_DISTANCE {
+        if self.last_key.is_empty() || self.index_handle.size >= DEFAULT_PROP_SIZE_INDEX_DISTANCE {
```

Comment:
why we still use DEFAULT value here? 
should we use the configuration now?

Reply:
SizeProperties is only for compatibility, we use RangeProperties instead after v2.1